editor/src/components/editor/editor-component.tsx

```diff
@@ -139,6 +142,24 @@ export const EditorComponentInner = React.memo((props: EditorProps) => {
 
   const setClearKeyboardInteraction = useClearKeyboardInteraction(editorStoreRef)
 
+  const mode = useEditorState(Substores.restOfEditor, (store) => store.editor.mode, 'mode')
+  React.useEffect(() => {
+    if (mode.type === 'live') {
+      dispatch([
+        EditorActions.showToast(
+          notice(
+            'You are in Live mode. Use ⌘ to select and scroll.',
+            'NOTICE',
+            true,
+            liveModeToastId,
+          ),
+        ),
+      ])
+    } else {
+      dispatch([EditorActions.removeToast(liveModeToastId)])
+    }
+  }, [mode, dispatch])
```

Comment:
I think you only care about running this if `mode.type` changes right?

Reply:
oh right!

Reply:
https://github.com/concrete-utopia/utopia/pull/3746/commits/b045856db447d90aed7c1cfe022ea049c61b5002